server/src/pages/delete-account/server.js

```diff
@@ -22,11 +22,11 @@ app.post("/delete", function (req, res) {
   }
   // FIXME: this just gets deleted due to ON DELETE CASCADE
   // Maybe we should keep but anonymize device_activity on account deletion
-  addDeviceActivity(req.deviceId, "delete-account");
+  //addDeviceActivity(req.deviceId, "delete-account");
```

Comment:
I did intend to, the FIXME describes the problem, which I thought was innocuous but was showing up in the logs